lib/js/components/Banner/Banner.vue

```diff
@@ -291,6 +244,41 @@
 	&__expandedText {
 		margin-top: $space-xs;
 	}
+
+	&.-warning {
```

Comment:
I think modifying classes of main component class should be after it's rules, not at the end after children rules.